src/dune_rules/pkg_rules.ml

```diff
@@ -652,7 +759,7 @@ module Run_with_path = struct
 
     let is_useful_to ~memoize:_ = true
 
-    let encode { prog; args; ocamlfind_destdir } path _ : Dune_lang.t =
+    let encode { prog; args; ocamlfind_destdir; _ } path _ : Dune_lang.t =
```

Comment:
I would suggest to specifically state the arguments that are being `_`'ed in such cases; otherwise you opt out of useful compiler errors that will tell you that you haven't dealt with an argument. Even if the case you have now might be legit, it will make it harder for future modifications.

(But in any case I believe `pkg_name` should also be encoded)